improver/cli/construct_reliability_tables.py

```diff
@@ -51,21 +53,28 @@ def process(
             calibration table using summation. This is equivalent to constructing
             then using aggregate-reliability-tables but with reduced memory
             usage due to avoiding large intermediate data.
+        cycle_point (str):
+            Current cycle point. Used in combination with max_days_offset to identify
+            which historic forecasts and truths to use.
+        max_days_offset (int):
+            Maximum offset in days, used to identify the oldest acceptable inputs
+
 
     Returns:
         iris.cube.Cube:
             Reliability tables for the forecast diagnostic with a leading
             threshold coordinate.
     """
-    from improver.calibration import split_forecasts_and_truth
     from improver.calibration.reliability_calibration import (
-        ConstructReliabilityCalibrationTables,
+        MetaConstructReliabilityCalibrationTables,
     )
 
-    forecast, truth, _ = split_forecasts_and_truth(cubes, truth_attribute)
-
-    return ConstructReliabilityCalibrationTables(
+    return MetaConstructReliabilityCalibrationTables(
+        truth_attribute=truth_attribute,
         n_probability_bins=n_probability_bins,
         single_value_lower_limit=single_value_lower_limit,
         single_value_upper_limit=single_value_upper_limit,
-    )(forecast, truth, aggregate_coordinates)
+        aggregate_coordinates=aggregate_coordinates,
+        cycle_point=cycle_point,
+        max_days_offset=max_days_offset,
+    )(forecast_directory, truth_directory)
```

Comment:
Hmmmm, there is a lot to unpack here in terms of the implications going forwards.
Our future vision actually separates loading, polling, file identity from the IMPROVER plugins.
In this vision, I the existing IMPROVER plugins wouldn't be modified to identify the relevant files.  Instead a plugin would specifically handle this and feed into the existing IMPROVER plugins.

We should chat offline.